### PR TITLE
Check path variables for cartridges in package.json + added output channel for debugging

### PR DESCRIPTION
**Pull request by taurgis**
With the new mobile first setup referencing the main catridge in the package "paths" parameter I have added support for this. This allows you to browse the main sfcc cartridge in a different workspace when creating a plugin for example.

Also added an output channel to allow bugreports to include output.

### CODE REVIEW

File: src/lib/CartridgeHelper.ts [hunk 1]
```diff
@@ -49,6 +51,63 @@ export const toCardridge = (projectFile: string, activeFile?: string): Promise<C
...
+ * @param workspaceRoot The current workspaceroot
+ * @param packageFile The path to the package file.
+ */
+export const getPathsCartridges = (workspaceRoot, packageFile): Promise<string[]> => {
+	return new Promise((resolve) => {
+		pathExists(packageFile).then(packageExists => {
+			if (packageExists) {
+				readFile(packageFile, 'UTF-8', (error, data) => {
+					if (error) {
+						resolve();
```
Comment (SqrTT) [inline src/lib/CartridgeHelper.ts]: Error cases should be rejected

**Comment by SqrTT** [comment]
Probably I missed something but I can't find `paths` property in `package.json` in [MFSG](https://bitbucket.org/demandware/mobile-first-reference-architecture/src/ecac4e7184512a847f6945f572a8425c0205da20/package.json?at=master&fileviewer=file-view-default)

Is the property specific for your project?

**Comment by taurgis** [comment]
No this is a general thing for plugins and projects to expand mfsg that is recommended by the mobile architecture like here:

https://bitbucket.org/demandware/plugin-applepay/src/60e6cbcf8a068bf18e378b9444e493bbe14c60d7/package.json?at=master&fileviewer=file-view-default

File: src/lib/CartridgeHelper.ts (hunk 1 above)
Comment (taurgis) [inline src/lib/CartridgeHelper.ts]: Will look into it asap.

**Comment by taurgis** [comment]
But we also use it in our project yes.

File: src/lib/CartridgeHelper.ts (hunk 1 above)
Comment (taurgis) [inline src/lib/CartridgeHelper.ts]: Reject cases have been added

**Comment by SqrTT** [comment]
Sorry for the long delay. Deadlines are coming :)

**Pull request merged**